autoload/neoformat/formatters/typescript.vim

```diff
@@ -1,5 +1,5 @@
 function! neoformat#formatters#typescript#enabled() abort
-   return ['tsfmt', 'prettier', 'tslint', 'eslint_d', 'clangformat', 'denofmt']
+   return ['tsfmt', 'prettier', 'prettier-eslint', 'tslint', 'eslint_d', 'clangformat', 'denofmt']
```

Comment:
not positive, but I think this has to be the same string as the function, so in this case it would be `prettiereslint` instead of `prettier-eslint`

```suggestion
   return ['tsfmt', 'prettier', 'prettiereslint', 'tslint', 'eslint_d', 'clangformat', 'denofmt']
```

Reply:
uhh... true, sorry for the inconveniences